packages/agoric-cli/src/commands/inter.js

```diff
@@ -0,0 +1,376 @@
+ * @param {import('agoric/src/lib/format.js').AssetDescriptor[]} assets
+ */
+export const fmtBid = (bid, assets) => {
+  const fmt = makeFormatters(assets);
+
+  const { offerArgs } = bid;
+  /** @type {{ price: string } | { discount: number }} */
+  const spec =
+    'offerPrice' in offerArgs
+      ? { price: fmt.price(offerArgs.offerPrice) }
+      : { discount: fmt.discount(offerArgs.offerBidScaling) };
+
+  const {
+    id,
+    error,
+    proposal: { give },
+    offerArgs: { want },
+    payouts,
+  } = bid;
+  const props = {
+    ...(give ? { give: fmt.record(give) } : {}),
+    ...(want ? { want: fmt.amount(want) } : {}),
+    ...(payouts ? { payouts: fmt.record(payouts) } : {}),
+    ...(error ? { error } : {}),
+  };
+  return harden({ id, ...spec, ...props });
+};
+
+/**
+ * @param {{
+ *   env: Partial<Record<string, string>>,
+ *   stdout: Pick<import('stream').Writable,'write'>,
+ *   stderr: Pick<import('stream').Writable,'write'>,
+ *   clock: () => number,
+ *   createCommand: // Note: includes access to process.stdout, .stderr, .exit
+ *     typeof import('commander').createCommand,
+ *   execFileSync: typeof import('child_process').execFileSync
+ * }} process
+ * @param {{ fetch: typeof window.fetch }} net
+ */
+export const makeInterCommand = async (
+  { env, stdout, stderr, clock, execFileSync, createCommand },
+  { fetch },
+) => {
+  const interCmd = createCommand('inter')
+    .description('Inter Protocol tool')
+    .option('--home [dir]', 'agd CosmosSDK application home directory')
+    .option(
+      '--keyring-backend [os|file|test]',
+      'keyring\'s backend (os|file|test) (default "os")',
+    );
+
+  const rpcTools = async () => {
+    const networkConfig = await getNetworkConfig(env).catch(err => {
+      throw Error(
+        `cannot get network config (${env.AGORIC_NET || 'local'}): ${
+          err.message
+        }`,
+      );
+    });
+    const { agoricNames, fromBoard, vstorage } = await makeRpcUtils(
+      { fetch },
+      networkConfig,
+    ).catch(err => {
+      throw new CommanderError(
+        1,
+        'RPC_FAIL',
+        `RPC failure (${env.AGORIC_NET || 'local'}): ${err.message}`,
+      );
+    });
+    const unserializer = boardSlottingMarshaller(fromBoard.convertSlotToVal);
+    return { networkConfig, agoricNames, fromBoard, unserializer, vstorage };
+  };
+
+  const bigintReplacer = (k, v) => (typeof v === 'bigint' ? `${v}` : v);
+  const pprint = v => JSON.stringify(v, bigintReplacer, 2);
+
+  const liquidationCmd = interCmd
+    .command('liquidation')
+    .description('liquidation commands');
+  liquidationCmd
+    .command('status')
+    .description(
+      `show amount liquidating, oracle price
+
+For example:
+
+{
+  "liquidatingCollateral": "10IbcATOM",
+  "liquidatingDebt": "120IST",
+  "price": "12.00 IST/IbcATOM"
+}
+`,
+    )
+    .option('--manager [number]', 'Vault Manager', Number, 0)
```

Comment:
I could see someone forgetting to specify the manager, but I suppose the offer will fail quickly anyway because the type guard will see that the give is the wrong brand